modules/prometheus-exporter_oracle/conf/02-dbisdown.yaml

```diff
@@ -0,0 +1,21 @@
+module: oracle
+name: "Database status"
+id: dbisdown
+
+transformation: true
+aggregation: ".mean(by=['dbname','dbtype','hostname'])"
+
+signals:
+  signal:
+    metric: oracledb_up
+    rollup: latest
```

Comment:
Can you explain to me what is for ? if you have "concurrent" datapoint for this MTS : `oracledb_up[dbname,dbtype,hostname']` perhaps it is better to simply not aggregate at all here (or add a dimension a source level if it miss something to differenciate them)

Reply:
Yes I can drop the aggrega and let the dimension manage at the source level. In fact, it os more readable to have a clear description in the incident such as host / db / dbtype, but basically all metrics could works with dimensions which are managed at the source level. 
=> I will drop aggegra for all metrics, it will be less "human readable" but will avoid any issues related to concurrent datapoint.

Reply:
no matter you aggregate or not, all dimensions are available in the alert title

Reply:
yes, but it's a little non friendly and "clear" like below test : 

`Alert:
[Critical][sandbox-sosdba] Oracle alert.log errors count error(s) detected > 0 (0.0)

Cleared at:
2021-11-15T16:30Z

Dimensions:
{instance=localhost:9161, scheme=http, service.name=oracle-exporter-1, erreurora=ORA-00257, env=sandbox-sosdba, sfx_monitored=true, hostname=ora-batman, dbname=SFXCDB, port=9161, sf_metric=oracledb_AlertLogError_count, dbtype=container_DB, host.name=localhost, job=oracle-exporter-1}
`

independently I drop the aggregation :)

Reply:
yes you get all dimensions if you do not aggregate which could be a bit long but at least it works in all cases and if you give the user a variable to modify the aggregation he still can aggregate himself depending on its own (custom) dimensions if he want more concise alert title